examples/04_scalability.py

```diff
@@ -197,3 +201,49 @@ def wrapped_func(*args, **kwargs):
 [t.set(size=16) for t in ax2.get_yticklabels()]
 plt.tight_layout()
 
+################################################################################
+# Similarity encoder with most frequent strategy and float32 dtype
+# ----------------------------------------------------------------
+#
+# The most frequent strategy selects the n most frequent values in a dirty
+# categorical variable to reduce the dimensionality of the problem and thus
+# speed things up. We select manually the number of prototypes we want to use.
```

Comment:
The sentences right above seem redundant with what was said before. I am not sure that they are necessary.